DashAI/back/preprocessor/column_dropper_by_name.py

```diff
@@ -0,0 +1,54 @@
+from beartype import beartype
+from datasets import DatasetDict
+
+from DashAI.back.dataloaders.classes.dashai_dataset import DashAIDataset
+from DashAI.back.preprocessor.base_converter import BaseConverter
+
+
+class ColumnDropperByName(BaseConverter):
+    """Converter to drop columns from the dataset"""
+
+    @beartype
+    def __init__(self, columns: list[str] | str):
```

Comment:
Why not call the parameter in a more descriptive way, such as `column_names`?

```suggestion
    def __init__(self, column_names: list[str] | str):
```